docs/release-notes/0.7/release-0.7.md

```diff
@@ -0,0 +1,142 @@
+# ML.NET 0.7 Release Notes
+
+Today we are excited to release ML.NET 0.7, which our algorithms strongly
+recommend you to try out! This release enables making recommendations with
```

Comment:
Was there a probability associated with this recommendation/prediction? We should create docs showing how to train this model using the new `Matrix Factorization` to create a recommendation model.